build.sbt

```diff
@@ -11,6 +11,7 @@ crossScalaVersions := Seq("2.11.8", "2.12.8", "2.13.1")
 libraryDependencies ++= Seq(
     "com.typesafe.akka" %% "akka-actor" % "2.6.0" % "provided",
     "org.quartz-scheduler" % "quartz" % "2.3.2" exclude ("com.zaxxer", "HikariCP-java7"),
+    "net.redhogs.cronparser" % "cron-parser-core" % "3.5",
```

Comment:
See my comment on the root of this PR, but I don't think we should add additional libraries just to improve logging a bit.

Reply:
I don't agree with "improve logging a bit".

Reply:
>> this library also pulls in specific versions of slf4j, apache commons, and joda time
Indeed, I somehow overlooked these dependencies....

Reply:
Right. Do you have a way of doing this without pulling in another library?